mistaken-pull-closer.yml

```diff
@@ -0,0 +1,18 @@
+# The JSONPath filter expression used to identify which PRs to close.
+# The data filtered is the pull request data along with other metadata passed in
+# by probot.
+# See http://goessner.net/articles/JsonPath/
+# `true` will close all PRs.
+filters:
+    - true
+
+# The message to post to the closed PR.
+commentBody: |
+  Thanks for your contribution!  Unfortunately, this GitHub repository is a
+  mirror of [http://source.android.com] and cannot be used for accepting
```

Comment:
This gets rendered as markdown, right?  If so, use angle brackets to have the URLs turned into links.... `<https://source.android.com>` (also "https").

Additionally, do you know if these line breaks will be interpreted literally?  I know if you put a line break in the GitHub comment box, it will be rendered that way, but I'm not sure if the yaml parser will recombine these into a single string.  If not, you may need to combine these to a single string.  Do you have a pointer where you tested this in one of the test orgs?

Reply:
Line breaks interpreted literally:
https://github.com/cco3/mistaken-pull-closer/pull/3#issuecomment-435988216

Will fix.